packages/specs/schema/src/components/default/genericsMapper.ts

```
@@ -1,70 +1,97 @@
+function getNestedSchema(propertyKey: string, schema: JsonSchema | JsonLazyRef, options: JsonSchemaOptions): JSONSchema7 | null {
+  try {
+    if ("isLazyRef" in schema) {
+      return null;
+    }
 
-  if (generics && obj.$ref) {
-    if (generics.has(obj.$ref)) {
-      let type = generics.get(obj.$ref);
+    if (schema.isGeneric) {
+      const genericLabel = schema.get(VendorKeys.GENERIC_LABEL);
+      let [model, next]: [Type | JsonSchema, any] = ([] as any).concat(options.generics?.[genericLabel]);
```

Biome (1.9.4)

[error] This let declares some variables that are only assigned once.

'model' is never reassigned.

'next' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)